tmt.spec

```diff
@@ -44,6 +44,7 @@ BuildRequires: python%{python3_pkgversion}-pytest
 BuildRequires: python%{python3_pkgversion}-click
 BuildRequires: python%{python3_pkgversion}-fmf
 BuildRequires: python%{python3_pkgversion}-mock
+BuildRequires: python%{python3_pkgversion}-nitrate
```

Comment:
So this does not work/help because of the missing config. Should be removed.

Reply:
Reverted by 31646f8